uni/android/app/build.gradle

```diff
@@ -32,7 +32,7 @@ if (keystorePropertiesFile.exists()) {
 }
 
 android {
-    compileSdkVersion 33
+    compileSdkVersion flutter.compileSdkVersion 
```

Comment:
Is this change needed? The sdk version was being bumped to suppress warnings.

Reply:
I think it's better, considering that newer versions of flutter will solve this warning if i'm not mistaken and removes the need to manually update this every year.

Reply:
Maybe, but perhaps that should be done in a PR focused on chore.

Reply:
done